ulmo/waterml/common.py

```diff
@@ -139,7 +139,7 @@ def _element_dict(element, exclude_children=None, prepend_attributes=True):
     element_dict.update(dict([
         (_element_dict_attribute_name(key, element_name,
             prepend_element_name=prepend_attributes), value)
-        for key, value in element.attrib.iteritems()
+        for key, value in element.attrib.items()
```

Comment:
And here we can see the use of automated tools :wink: 

The first pass changed `iteritems` to `item`, if you pass it again it will recommend to add the `list`.  (Which is not needed, see my comment below.)